examples/examples_py_test.go

```diff
@@ -0,0 +1,47 @@
+//go:build python || all
+// +build python all
+
+package examples
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/pulumi/pulumi/pkg/v3/testing/integration"
+)
+
+func getPythonBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	base := getBaseOptions(t)
+	basePython := base.With(integration.ProgramTestOptions{
+		Dependencies: []string{
+			filepath.Join("..", "sdk", "python", "bin"),
+		},
+	})
+
+	return basePython
+}
+
+// func TestPyExamples(t *testing.T) {
```

Comment:
I saw you updated the python example too. Should we uncomment this to run it?

Reply:
I wasn't sure how the python examples get the right version of the package. I couldn't get it to pull the right version locally without a `-e ../sdk/python` and I didn't want to leave that in the example. If it should `just work` then I am good to test it as well.

Reply:
I think maybe line 17 needs to be `filepath.Join("..", "sdk", "python", "pulumi_kubernetes_ingress_nginx")`? 

This is always a little tricky to get right! I'm fine with leaving it disabled if you can't work it out.

Reply:
Cool, thank you. I will check it out, and let you know early next week.

Reply:
I tried a few different things. I am going to let this go in, release, then re-enable the python test.